jea/audit-and-report.md

```diff
@@ -134,14 +160,14 @@ PS>CommandInvocation(Get-Service): "Get-Service"
 Running  Dns                DNS Server
 ```
 
-For each command a user runs, a "CommandInvocation" line will be written, describing the cmdlet or function the user invoked.
-ParameterBindings follow each CommandInvocation to tell you about each parameter and value that was supplied with the command.
-In the above example, you can see that the parameter "Name" was supplied the value "Dns" for the "Get-Service" cmdlet.
+A **CommandInvocation** line is written for each command a user runs. **ParameterBindings** record
+each parameter and value supplied with the command. In the previous example, you can see that the
+parameter **Name** was supplied the with value **Dns** for the `Get-Service` cmdlet.
 
-The output of each command will also trigger a CommandInvocation, usually to Out-Default.
-The InputObject of Out-Default is the PowerShell object returned from the command.
-The details of that object are printed a few lines below, closely mimicking what the user would have seen.
+The output of each command also triggers a **CommandInvocation**, usually to `Out-Default`. The
+**InputObject** of `Out-Default` is the PowerShell object returned from the command. The details of
+that object are printed a few lines below, closely mimicking what the user would have seen.
 
 ## See also
 
-- [*PowerShell ♥ the Blue Team* blog post on security](https://blogs.msdn.microsoft.com/powershell/2015/06/09/powershell-the-blue-team/)
+[*PowerShell ♥ the Blue Team* blog post on security](https://blogs.msdn.microsoft.com/powershell/2015/06/09/powershell-the-blue-team/)
```

Comment:
This link resolves to https://devblogs.microsoft.com/powershell/powershell-the-blue-team/

Reply:
fixed